src/components/header/nav.tsx

```diff
@@ -1,64 +1,76 @@
 import { Box, Link } from "@mui/material";
 import { Auth } from "../../atoms/auth";
+import { NavButtonProps } from "../../types";
 
-interface HomeNavProps { auth: Auth | undefined}
-
-const HomeNav = (props: HomeNavProps) => {
-    const { auth } = props;
+/**
+ * Component props
+ * @date 9/18/2023 - 4:04:33 PM
+ *
+ * @interface Props
+ * @typedef {Props}
+ */
+interface Props {
+  auth: Auth | undefined;
+}
 
-    interface NavButtonProps {
-        text: string, 
-        selected: boolean, 
-        sx_props: object
-    } 
-    const NavButton = (props: NavButtonProps) => {
-      const {text, selected, sx_props} = props;
-      return (
-        <Link 
-            href="#"
-            underline="none"
-            sx={ Object.assign({
-          "background-color": (selected ? "#dee2e5" : ""),
-          "height": "100%",
-          "padding": "15px"
-        }, sx_props)}>
-          { text }
-        </Link>
-      );
-    }
+const HomeNav = ({ auth }: Props) => {
 
-    return (
-        <Box sx={{
-            width:"100%",
-            display:"flex",
-            justifyContent:"space-between",
-            "padding-top": "16px",
-            "padding-right": "22px",
-            "padding-bottom": "14px",
-            "padding-left": "0px"
-          }}>
-            <Box>
-              <NavButton 
-                  text="Home" 
-                  selected={ true } 
-                  sx_props={{
-                "border-top-left-radius": "15px",
-                "border-bottom-left-radius": "15px"
-              }}/>
-              <NavButton text="Admin" selected={false} sx_props={{}}/>
-              <NavButton text="On call" selected={false} sx_props={{}}/>
-            </Box>
-            <Box>
-              <Link href="#" type="button" onClick={() => auth?.logout()} sx={{
-                display:"flex",
-                justifyContent:"right"
-              }}>
-                Log out
-              </Link>
+  const NavButton = ({ text, selected, sx_props }: NavButtonProps) => (
```

Comment:
JSDoc commments here.
Because of the way you are using this component I would recommend exporting the NavButton to its own file as its own component and then importing it in here. Buuuut maybe this way is also fine, your call

Reply:
new file added